packages/language-server/src/server.ts

```diff
@@ -86,9 +86,9 @@ export function startServer(options?: LSOptions) {
     let sveltePlugin: SveltePlugin = undefined as any;
 
     connection.onInitialize((evt) => {
-        const workspacePath = urlToPath(evt.rootUri || '') || '';
-        Logger.log('Initialize language server at ', workspacePath);
-        if (!workspacePath) {
+        const workspacePaths = evt.workspaceFolders?.map(folder => folder.uri.toString()) ?? [];
```

Comment:
For potential backward compatible and for LSP clients that don't support multi-root-workspaces can you make it fallback to the old behavior?

Reply:
Fallback to `evt.rootUri`?

Reply:
Yes.